portfolio project final version - All files included index, css style sheets and all related files

baqar-abbas opened the pull request:
Portfolio project final version - All files included index.html, styles.css style sheets, linters configurations files, read me file and all related files of the Project including all image files. All files added to final feature Branch.

please share your valuable feedback and kindly approve as I need to submit my project before 7 P.m according to the program time.

baqar-abbas committed:
Linters Configurations, added gitignore file, added package json

Linters Configurations, added gitignore file, added package json and updated readme file.

baqar-abbas committed:
Index.html and styles.css files created and saved

Project's index.html and styles.css files created updated and saved in finalfeature branch

baqar-abbas committed:
Hamburger menu and header background Images updated in finalfeature

Hamburger menu and header and footer background images added to finalfeature branch

baqar-abbas committed:
Tonic project image file added

Tonic Project Image added to final feature branch

baqar-abbas committed:
Multi post Series Project Image added

Multi post Series Project Image added to finalfeature branch

baqar-abbas committed:
facebook 360 project Image file added

Facebook 360 project Image file added to finalfeature branch

baqar-abbas committed:
Uber navigation project Image file added

Uber navigation project Image file added to finalfeature branch`

baqar-abbas committed:
HTML 5 Image file added

HTML 5 Image file added to final feature branch

baqar-abbas committed:
CSS3 image file added to the branch

CSS3 image file added to the final feature branch branch

baqar-abbas committed:
javascript image file added to the branch

javascript image file added to the final feature branch

baqar-abbas committed:
Eclipse image file added to the branch

Eclipse image file added to the final feature branch

KanzaTahreem reviewed (commented):
# INVALID ❌ 

Hi @baqar-abbas  👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌

- [ ] Unfortunately,  you have created a new repository which does not include pull requests done previously by other reviewers. I can see that your profile has another repository for portfolio project where [the reviewer asked you to submit a valid PR](https://github.com/baqar-abbas/Portfolio_Project/pull/3). However, instead of doing that you created a new repository. 

- [ ] Please ensure that in the second day at Microverse, we learned that we should use the same PR we used for the the first review until approval. You can take a step back and refresh your knowledge about the the code review flow at Microverse. Please follow the lessons mentioned below for more clarity 🔽

- [How to ask for a code review at Microverse](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md)
- [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md)
- [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [ ] I think it would be better if you [proceed according to the previous reviewers's instructions ](https://github.com/baqar-abbas/Portfolio_Project/pull/3) and do not create a new repository for submitting a PR.

Please ensure that you professionally document your repository.


_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_**Reviewed By [Kanza Tahreem](https://github.com/KanzaTahreem)**_

KanzaTahreem commented:
Hi @baqar-abbas 👋
Please proceed and submit this PR again. Good luck 👍

## **_For reviewer_** 🔽
I noticed that the student created a new repository for the portfolio project. However, according to the [code review flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md) we should use the same PR we used for the the first review until approval. **But this PR is an exception.** 🎯
Please ensure that we have a conversation before you proceed with your remarks. I already had a conversation with seniors, so I think I can guide you well with the process ✌

baqar-abbas commented:
Hi Kanza, Thanks for your guidance I have submitted the below mentioned Pull request for review again.
https://github.com/baqar-abbas/Portfolio-Project-Final-version/pull/1

baqar-abbas commented:
This is the fourth time in last 7 hours I am submitting my request (which were marked invalid) please get it reviewed this time.

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+body {
+    font-family: 'Poppins';
+}
+</style>
```
Kindly take all your stylings to an external sheet🤗 it is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that we [keep to HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)🤗
<img width="463" alt="cssrule" src="https://user-images.githubusercontent.com/64643995/212496063-0e09f380-40a8-4143-907f-f6ba26e07e00.png">

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+      <nav>
+        <ul>
+          <li><a class="logolink" href="">My Logo</a></li>
+          <li style="float: right;"><a href=""><img src="hammenu.png" alt="ham" style="width: 45px;height: 45px;"></a></li>
```
Kindly take all your stylings to an external sheet🤗 it is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that we [keep to HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)🤗
<img width="463" alt="cssrule" src="https://user-images.githubusercontent.com/64643995/212496063-0e09f380-40a8-4143-907f-f6ba26e07e00.png">

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+            <p>I’m a software developer! I can help you build a product , feature or website Look <br> through 
+              some of my work and experience! If you like what you see and have a project <br>
+              you need coded, don’t hestiate to contact me.</p>
+              <p style="color: #7F8CFF;">LET’S CONNECT</p>
```
Kindly take all your stylings to an external sheet🤗 it is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that we [keep to HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)🤗
<img width="463" alt="cssrule" src="https://user-images.githubusercontent.com/64643995/212496063-0e09f380-40a8-4143-907f-f6ba26e07e00.png">

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+</style>
+    </head>
+    <body>
+      <nav>
+        <ul>
+          <li><a class="logolink" href="">My Logo</a></li>
+          <li style="float: right;"><a href=""><img src="hammenu.png" alt="ham" style="width: 45px;height: 45px;"></a></li>
+        </ul>
+      </nav>
```
For semantic purposes, I recommend Your `<nav>` should be wrapped inside your `<header>` tag. this aids accessibility for screen readers🤗

Ibizugbe commented on styles.css:
```diff
@@ -0,0 +1,417 @@
+nav li a {
+  text-decoration: none;
+  color: blue;
+  display: inline-block;
```
It is a requirement that your `<nav>` layout should be done with flexbox, kindly fix this 🤗

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+                <p class="popp13">Uber Navigation * <span class="popp13">Lead Developer</span> * <span class="popp13">2018</span></p>
+                <br>
+                <p class="popp15">A smart assistant to make driving more safe, efficient, and fun <br> by unlocking 
+                    your most expensive computer: your car..</p><br>
+                <span class="htcj4">HTML</span> 
+                <span class="htcj4">Ruby on Rails</span>
+                <span class="htcj4">CSS</span>
+                <span class="htcj4">Javascript</span><br><br>
+                  <div class="btn4">
+                    <a href="image4.png" target="_blank">See Project</a>
+                  </div> 
+                  </div>
+                  <div class="imgdiv4">
+                    <img src="image4.png" alt="Nature" height="300" width="300">
+                </div>
+        </div>    
+       </div>  
+       <div class="container5">
+        <div class="about">
+            <h1 class="abouth">About Myself</h1>
+            <p class="popp15">Hello I’m a software developer! I can help you build a <br> product , feature or 
+                website Look through some of my <br> work and experience! If you like what you see and
+                 have <br> a project you need coded, don’t hestiate to contact me.</p>
+             <span class="popp15">Let's Connect</span>   
+             <section><i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-linkedin-in"></i>
+              <i class="fa-solid fa-m"></i>
+              <i class="fa-brands fa-github"></i>
+              <i class="fa-regular fa-hand-peace"></i></section><br><br>
+                     <span><a class="abouts" href="#">Get My Resume</a> </span> 
+        </div>
+        <div class="language">
+            <p class="aboutp">Languages</p>
+            <img src="js.png" alt="Js" width="100px" height="100px">
+            <img src="ht.png" alt="Js" width="100px" height="100px">
+            <img src="css.png" alt="Js" width="100px" height="100px">
+            <p class="aboutp">Frameworks</p>
+            <p class="aboutp">Skills</p>
+        </div>
+       </div>
```
It is a project requirement that you have major sections, the work section and the about section🤗 Kindly wrap all your projects in a `<section>`  and your  contents about you in a `<section>`🤗 also, your major sections should be aligned with css grid while it's content should be aligned with flexbox😉

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+                  </div> 
+                  </div>
+                  <div class="imgdiv2">
+                    <img src="image2.png" alt="Nature" height="300" width="300">
```
kindly take your styles to an external stylesheet as requested previously🤗

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+              <i class="fa-regular fa-hand-peace"></i></section>
+            </div>
+        </header>
+       <div class="container">
+        <div class="imgdiv">
+            <img src="image1.png" alt="Nature" height="300" width="300">
+        </div>
+        <div class="tonic">
+                <h1 class="popheading">Tonic</h1>
+                <p class="popp13">CANOPY * <span class="popp13">Back End Dev</span> * <span class="popp13">2015</span></p>
+                <br>
+                <p class="popp15">A daily selection of privately personalized reads; no accounts <br> or sign-ups required.</p>
```
your project cards are appearing different from the design, kindly ensure you are adhering strictly to the design😉
your project
<img width="258" alt="yourApp" src="https://user-images.githubusercontent.com/64643995/212498591-0e109cfa-c5c7-4fa2-9e88-4ea5163b8c30.png">

the design
<img width="187" alt="the-design" src="https://user-images.githubusercontent.com/64643995/212498609-c6027974-79a5-465c-b026-757369a4a5f4.png">

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+                <span class="htcj">HTML</span> 
+                <span class="htcj">CSS</span>
+                <span class="htcj">Javascript</span><br><br>
+                  <div class="btn">
+                    <a href="image1.png" target="_blank">See Project</a>
+                  </div>
```
You haven't implemented the states for your buttons, kindly ensure the different states are implemented as given in the design🤗
<img width="348" alt="btnstate" src="https://user-images.githubusercontent.com/64643995/212498712-53fef01b-8916-4a96-8400-3b216d27c1cf.png">

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,150 @@
+            <h1 class="abouth">About Myself</h1>
+            <p class="popp15">Hello I’m a software developer! I can help you build a <br> product , feature or 
+                website Look through some of my <br> work and experience! If you like what you see and
+                 have <br> a project you need coded, don’t hestiate to contact me.</p>
+             <span class="popp15">Let's Connect</span>   
+             <section><i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-linkedin-in"></i>
+              <i class="fa-solid fa-m"></i>
+              <i class="fa-brands fa-github"></i>
+              <i class="fa-regular fa-hand-peace"></i></section><br><br>
+                     <span><a class="abouts" href="#">Get My Resume</a> </span> 
+        </div>
+        <div class="language">
+            <p class="aboutp">Languages</p>
+            <img src="js.png" alt="Js" width="100px" height="100px">
+            <img src="ht.png" alt="Js" width="100px" height="100px">
+            <img src="css.png" alt="Js" width="100px" height="100px">
+            <p class="aboutp">Frameworks</p>
+            <p class="aboutp">Skills</p>
+        </div>
```
I think your about section is also deviating from the given design, kindly take a look at the[ given design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=sAjeAygiiy8UdgKo-0) and ensure you stick to it.🤗

your app
<img width="245" alt="desin-abt" src="https://user-images.githubusercontent.com/64643995/212498812-75bde6ad-4d94-4e84-ba77-1e4147940e2c.png">

the design
<img width="180" alt="your-about" src="https://user-images.githubusercontent.com/64643995/212498821-4f753631-56f7-4655-9c56-49d6d333b28f.png">

Ibizugbe reviewed (changes requested):
Hi @baqar-abbas 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅
- linters are passing.
- the right github flow method has been used.



## Required Changes ♻️
- It is a requirement that you stick to the given design, I think your project currently has a couple of deviations from the[ design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=sAjeAygiiy8UdgKo-0), kindly compare [the design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=sAjeAygiiy8UdgKo-0) to your project and adjust all of them🤗🤗
- I suggest you take a proper look at the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) and ensure you are abiding to them😉
- Kindly ensure you stick to the font properties and spacing given in the design🤗
_kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baqar-abbas committed:
slight changes in Index.html file

baqar-abbas committed:
Import font weights in styles css

baqar-abbas committed:
Changed Required Updated

Index.html and styles.css files updated. Design layout adjusted as per the shared requirements by the Code Reviewer. All Inline CSS removed and stored in CSS file. New Design images added for a Perfect Figma Mobile layout Desgin.

baqar-abbas committed:
Linter errors removed and commit made

baqar-abbas commented:
Suggested Changes made in design and layout for mobile resolution. Now sending again for your kind review. Please share your valuable review.
Regards, Baqar Abbas

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+            <nav>
+                <ul>
+              <li><a class="logolink" href="">My Logo</a></li>
+              <li class="leftli"><a href=""><img src="hammenu.png" alt="ham" width="20px" height="18px"></a></li>
+                </ul>
```
- Just as the previous reviewer mentioned before, kindly refrain from adding inline styles but instead please add them in your CSS styles in your `styles.css` file. :100: 
- If you want to apply the width and height styles to the image then I think you should consider giving the image a `class `or an `id `. It could be something like this eg.
`<li class="leftli"><a href=""><img class ="imageOne"  src="hammenu.png" alt="ham" width="20px" height="18px"></a></li>
`- After this, you can then style your `imageOne` class in your `styles.css` file like so :
```
 .imageOne {
width = 20px
height = 18px
};
```

- This is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that must be adhered to in order to [keep to HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) so please make those changes to your project. :wink:

baqar-abbas replied:
All Images Height and width Attribute removed and added in CSS file. Sending again for review ...  
suggested changes made

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through 
+                  some of my work and experience! If you like what you see and have a project you need coded,
+                   don’t hestiate to contact me.</p>
+                  <p style="color: #7F8CFF;">LET’S CONNECT</p>
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css file`. :100:

baqar-abbas replied:
All inline CSS removed and added in CSS file. Sending again for review ...  
suggested changes made

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                   don’t hestiate to contact me.</p>
+                  <p style="color: #7F8CFF;">LET’S CONNECT</p>
+                <section>
+                    <img class="icons" src="twitter.svg" alt="twitter" width="17.99px" height="20px">
+                    <img class="icons" src="linkedinicon.svg" alt="linkedin" height="20px" width="22.99px" height="20px">
+                    <img class="icons" src="m.svg" alt="mimage" width="17.99px" height="20px">
+                    <img class="icons" src="github.svg" alt="github" width="17.99px" height="20px">
+                    <img class="icons" src="victory.svg" alt="victory" width="17.99px" height="20px">
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css file`. :100:
- [ ] I also noticed that your LinkedIn Image is broken so it doesn't display on the page. This could be a result of a spelling error with how the image was saved and how it was called as :wink: 
- It could also be that the file path was deleted etc. Kindly try to fix this as well. I have attached a screenshot below :100: 
![image](https://user-images.githubusercontent.com/102699958/213065847-7c45f598-8267-4d40-b709-fa5cd9b9982c.png)

baqar-abbas replied:
Mistakenly I added height attribute twice in the img tag now corrected and working fine 
![image](https://user-images.githubusercontent.com/117091923/213155250-ae88d7dc-14ac-45a5-85d9-3128ba955326.png)

baqar-abbas replied:
![image](https://user-images.githubusercontent.com/117091923/213155470-e47daed7-5ee9-4f46-bf43-089ab18f02e6.png)

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+            </div>    
+           </div>
+           <div class="container2">
+            <div class="imgdiv2">
+                <img src="image2.png" alt="Nature" height="220px" width="295px">
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css file`. :100:

baqar-abbas replied:
Suggested changes made all width and height attribute of images removed and stored in CSS files. Sending again for review.

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                </div>
+            </div>
+        <div class="container">
+            <div class="imgdiv">
+                <img src="image1.png" alt="Nature" height="220px" width="295px">
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css file`. :100:

baqar-abbas replied:
All inline CSS removed and stored in CSS file. Sending for review again

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+           </div>
+           <div class="container">
+            <div class="imgdiv">
+                <img src="image3.png" alt="Nature" height="300" width="300">
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css file`. :100:

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+           </div>
+           <div class="container2">
+            <div class="imgdiv2">
+                <img src="image4.png" alt="Nature" height="220px" width="295px">
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css file`. :100:

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                    <img class="icons" src="linkedinicon.svg" alt="linkedin" height="20px" width="22.99px" height="20px">
+                    <img class="icons" src="victory.svg" alt="victory" width="17.99px" height="20px">
+                    <img class="icons" src="twitter.svg" alt="twitter" width="17.99px" height="20px">
+                    <img class="icons" src="m.svg" alt="mimage" width="17.99px" height="20px">
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css file`. :100:
- [ ] I also noticed that your LinkedIn Image here is  also broken so it doesn't display on the page. This could be a result of a spelling error with how the image was saved and how it was called as :wink: 
- It could also be that the file path was deleted etc. Kindly try to fix this as well. I have attached a screenshot below :100: 
![image](https://user-images.githubusercontent.com/102699958/213066691-1d8bde6b-aca6-45a7-b43a-5b0f08affad9.png)

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                <ul>
+                    <li><p class="aboutp">Languages <span class="greater">&#8964;</span></p>
+                        <div class="javascript1">
+                            <img src="jss.png" alt="Js" width="50px" height="50px"> <span class="lastdiv">Javascript</span>
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css` file. :100:

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                            <img src="htt.png" alt="Js" width="50px" height="50px"> <span class="lastdiv">HTML</span>
+                        </div>
+                        <div class="javascript1">
+                            <img src="csss.png" alt="Js" width="50px" height="50px"> <span class="lastdiv">HTML</span>
```
- [ ] Kindly refer to my earlier comment about not adding inline styles in your HTML file but instead please add them in your CSS styles in your `styles.css` file. :100:

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+            </div>
+        <div class="container">
+            <div class="imgdiv">
+                <img src="image1.png" alt="Nature" height="220px" width="295px">
```
- [ ] I noticed that you didn't make use of the (`section` tags). The (`section` tag) is used to group together related content and give it an appropriate semantic heading. This is why it was great of you to wrap all your works in a section eg. (`Work-Container` ). :+1: It was also a part of the project requirements to wrap your sections in the (`section tags`) so kindly fix this as well. You can also read more about how the section tag is used [here](https://www.w3schools.com/tags/tag_section.asp) :smile: .
- In order to do this, kindly wrap this `div` and all the other `div tags` that contain the works section with the` class = "container, container2" ` etc. in a (`section` tag) :100:  like so:
```
<section>
<div class="container"></div>
<div class="container2"></div>
<div class="container"></div>
<div class="container2"></div>
</section>
```

- [ ] Please don't add `container5` and this is because it contains the About me section so that must also be wrapped in separate (`section` tags). :wink: 
```
<section>
<div class="container5"></div>
</section>
```

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
```
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") must be a list `ul` and each of them a `li`.
- UL and LI tags are used to create unordered lists in HTML. The UL tag defines an unordered list and the LI tag is used to define a list item within that unordered list. This allows for a clear and structured way to organize and display a list of items. Additionally, using these tags can make it easier to style the list using CSS :wink:

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
```
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") must be a list `ul` and each of them a `li`.
- UL and LI tags are used to create unordered lists in HTML. The UL tag defines an unordered list and the LI tag is used to define a list item within that unordered list. This allows for a clear and structured way to organize and display a list of items. Additionally, using these tags can make it easier to style the list using CSS :wink:

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
```
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") must be a list `ul` and each of them a `li`.
- UL and LI tags are used to create unordered lists in HTML. The UL tag defines an unordered list and the LI tag is used to define a list item within that unordered list. This allows for a clear and structured way to organize and display a list of items. Additionally, using these tags can make it easier to style the list using CSS :wink:

nahnahdev commented on styles.css:
```diff
@@ -0,0 +1,331 @@
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid gray;
```
- [ ] In the Figma design, this dark border around each of the cards had a different color than what you have here. Kindly try to follow the Figma design guidelines as much as possible :wink: 
- In Figma 
![image](https://user-images.githubusercontent.com/102699958/213072845-f9d8ce29-4111-40eb-bdd1-0bce04e7ec87.png)

nahnahdev commented on styles.css:
```diff
@@ -0,0 +1,331 @@
+  margin-right: auto;
+  width: 90%;
+
+  /* height: 600px; */
```
- [ ] Kindly avoid leaving commented pieces of code in your project. It is unprofessional and makes your code unclean. :+1:

nahnahdev commented on styles.css:
```diff
@@ -0,0 +1,331 @@
+  margin-right: auto;
+  width: 90%;
+
+  /* height: 600px; */
```
- [ ] Kindly avoid leaving commented pieces of code in your project. It is unprofessional and makes your code unclean. :+1:

baqar-abbas replied:
Commented piece of code removed

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                <p>I’m a software developer! I can help you build a product , feature or website Look through 
+                  some of my work and experience! If you like what you see and have a project you need coded,
+                   don’t hestiate to contact me.</p>
+                  <p style="color: #7F8CFF;">LET’S CONNECT</p>
```
- [ ] The title of this text must use a header tag ie. (`h1,h2,h3 ..)` and not a span tag. This is because it is a subtitle of the section, not a span text :100:

baqar-abbas replied:
changes made

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                <p class="popp15">Hello I’m a software developer! I can help you build a product , feature or 
+                    website Look through some of my work and experience! If you like what you see and
+                     have a project you need coded, don’t hestiate to contact me.</p>
+                 <span class="popp15">Let's Connect</span>   
```
- [ ] The title of this text must use a header tag ie. (`h1,h2,h3 ..)` and not a span tag. This is because it is a subtitle of the section, not a span text :100:

baqar-abbas replied:
Suggested changes made

nahnahdev commented on index.html:
```diff
@@ -0,0 +1,151 @@
+                    website Look through some of my work and experience! If you like what you see and
+                     have a project you need coded, don’t hestiate to contact me.</p>
+                 <span class="popp15">Let's Connect</span>   
+                 <section>
+                    <img class="icons" src="github.svg" alt="github" width="17.99px" height="20px">
+                    <img class="icons" src="linkedinicon.svg" alt="linkedin" height="20px" width="22.99px" height="20px">
+                    <img class="icons" src="victory.svg" alt="victory" width="17.99px" height="20px">
+                    <img class="icons" src="twitter.svg" alt="twitter" width="17.99px" height="20px">
+                    <img class="icons" src="m.svg" alt="mimage" width="17.99px" height="20px">
+                </section><br><br>
```
- [ ] The list of project technologies icons must be a list `ul` and each of them a `li`. SO kindly remove the section tags surrounding them and replace it with a `ul` and close each icon in an `li` tag :100: 
- UL and LI tags are used to create unordered lists in HTML. The UL tag defines an unordered list and the LI tag is used to define a list item within that unordered list. This allows for a clear and structured way to organize and display a list of items. Additionally, using these tags can make it easier to style the list using CSS :wink:

baqar-abbas replied:
Suggested changes made removed section and added UL tag and displaying images in LI tag. Sending for your review again kindly share your valuable review.

nahnahdev commented on styles.css:
```diff
@@ -0,0 +1,331 @@
+  color: #6070ff;
+}
+
+.btn,
+.btn3,
+.btn2,
+.btn4 {
+  border: 2px solid lightgray;
+  border-radius: 5px;
+  display: inline-block;
+  padding: 18px;
+}
```
- [ ] I also noticed that your buttons are missing the right hover effects but the `Get My Resume` button in the About section is missing all the effects. 
- [ ] Kindly ensure that you apply these styles while following the Figma design guidelines to improve user interactivity and also so your design can follow the design guidelines in the Figma template :+1: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213075707-0dcc2cef-b2d5-4fbf-a10f-61f5cf4eb11c.png)

baqar-abbas replied:
Hovering effects added to Get My resume Button. Sending for review again.

nahnahdev reviewed (changes requested):
Hi @baqar-abbas , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media3.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- Good use of GitHub flow :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baqar-abbas commented:
All the suggested changes have been incorporated. I am sending for review again kindly share your valuable remarks.

baqar-abbas committed:
Suggested Changes made by the code reviewer have been incorporated in HTML and Css files.

frank1738 reviewed (commented):
Hi @baqar-abbas , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with failing linters checks. Kindly fix them and resubmit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

baqar-abbas committed:
Spacing and indentation errors removed and saved

baqar-abbas commented:
Indentation errors removed and submitted again for review

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
```
- [ ] [REQUIRED] Please implement the [previous reviewer's](https://github.com/baqar-abbas/Portfolio-Project-Final-version/pull/1/files#r1073049076) suggestion 🙏🏼 The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul`, and each of them a `li`, not a `span` 😃

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
```
- [ ] [REQUIRED] Please implement the [previous reviewer's](https://github.com/baqar-abbas/Portfolio-Project-Final-version/pull/1/files#r1073049408) suggestion 🙏🏼 The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul`, and each of them a `li`, not a `span` 😃

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
```
- [ ] [REQUIRED] Please implement the [previous reviewer's](https://github.com/baqar-abbas/Portfolio-Project-Final-version/pull/1/files#r1073049510) suggestion 🙏🏼 The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul`, and each of them a `li`, not a `span` 😃

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
```
- [ ] [REQUIRED] Please implement the [previous reviewer's](https://github.com/baqar-abbas/Portfolio-Project-Final-version/pull/1/files#r1073049510) suggestion 🙏🏼 The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul`, and each of them a `li`, not a `span` 😃

baqar-abbas replied:
added technologies in UL and LI. Now sending for review again please approve its my third review

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                    <li><img class="icons" src="m.svg" alt="mimage"></li>
+                </ul>
+                <br><br>
+                         <span><a class="abouts" href="#">Get My Resume</a> </span> 
```
- [ ] [REQUIRED] Please implement the previous reviewer's suggestion about all button styles 🙏🏼 `[HINT]` Implement to your buttons like this style, actually add: `border: 1px solid #DFE1E6;` `border-radius: 16px;`

#
![image](https://user-images.githubusercontent.com/63915024/213191803-97f664c0-de7f-4fb2-b72c-06594ec976c1.png)
#
![image](https://user-images.githubusercontent.com/63915024/213192563-663402f7-f541-445a-b4ee-3d593fd122b6.png)
#

baqar-abbas replied:
Design adjusted as per suggestions. Now sending for review again please approve its my third review.

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                </ul>
+            </nav>
+        </header>
+        <div class="headline">
```
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                    </ul>
+                </div>
+            </div>
+        <div class="container">
```
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
```
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                      </div>
+            </div>    
+           </div>
+           <div class="container">
```
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
```
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                    
+            </div>    
+           </div>
+           <div class="container5">
```
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,152 @@
+            </div>    
+           </div>
+           <div class="container5">
+            <div class="about">
```
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼

mukhammadrizooff reviewed (changes requested):
Hi @baqar-abbas, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

<img src="https://media0.giphy.com/media/qLWdMYX1NYF2g/giphy-downsized-medium.gif"/>

 ## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baqar-abbas committed:
Suggested changes by the code reviewer made

baqar-abbas commented:
All the required (must do) changes have been incorporated. Optional suggestions I have noted down and will make them my practice in coming projects.
Now sending for review again please approve its my third review

jerryowusu reviewed (approved):
Hi @baqar-abbas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![clapping-leonardo-dicaprio](https://user-images.githubusercontent.com/65233860/213243630-72ab5885-d484-4b3c-885a-ed1b4dc3869f.gif)

## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baqar-abbas commented:
Thanks @jerryowusu I am Merging my branch into main branch
